lib/byron/src/Cardano/Wallet/Byron.hs

```diff
@@ -291,6 +297,7 @@ data Tracers' f = Tracers
     , apiServerTracer    :: f ApiLog
     , walletEngineTracer :: f (WorkerLog WalletId WalletLog)
     , walletDbTracer     :: f DBLog
+    , ntpClientTracer    :: f NtpTrace
```

Comment:
It doesn't necessarily need to be in its own tracer. It can also be rolled into another one if you think that might be better.

Reply:
I will leave it as it is now as separated, standalone-on-par-with-others tracer